sklearn/datasets/_species_distributions.py

```diff
@@ -239,6 +237,9 @@ def fetch_species_distributions(
            (b'microryzomys_minutus', -67.8   , -16.2667),
            (b'microryzomys_minutus', -67.9833, -15.9   )],
           dtype=[('species', 'S22'), ('dd long', '<f4'), ('dd lat', '<f4')])
+
```

Comment:
This reference can be removed because the example is already referenced in line 219.